contracts/fungible-tokens/andromeda-merkle-airdrop/src/testing/tests.rs

```diff
@@ -1,668 +1,697 @@
-            attr("amount", test_data.amount)
-        ]
-    );
-
-    // makes the stage expire
-    env.block.time = Timestamp::from_nanos(100_000_000 + 2);
-
-    // Can burn after expired stage
-    let msg = ExecuteMsg::Burn { stage: 1u8 };
-
-    let info = mock_info("owner0000", &[]);
-    let res = execute(deps.as_mut(), env, info, msg).unwrap();
-
-    let expected = SubMsg::new(CosmosMsg::Wasm(WasmMsg::Execute {
-        contract_addr: MOCK_CW20_CONTRACT.to_string(),
-        funds: vec![],
-        msg: to_json_binary(&Cw20ExecuteMsg::Burn {
-            amount: Uint128::new(9900),
-        })
-        .unwrap(),
-    }));
-    assert_eq!(res.messages, vec![expected]);
-
-    assert_eq!(
-        res.attributes,
-        vec![
-            attr("action", "burn"),
-            attr("stage", "1"),
-            attr("address", "owner0000"),
-            attr("amount", Uint128::new(9900)),
-        ]
-    );
-}
-
-#[test]
-fn test_can_burn_native() {
-    let mut deps = mock_dependencies_custom(&[coin(100000, MOCK_NATIVE_DENOM)]);
-    let test_data: Encoded = from_json(TEST_DATA_1).unwrap();
-
-    let msg = InstantiateMsg {
-        asset_info: Asset::NativeToken(MOCK_NATIVE_DENOM.to_string()),
-        kernel_address: MOCK_KERNEL_CONTRACT.to_string(),
-        owner: None,
-    };
-
-    let mut env = mock_env();
-    let info = mock_info("owner0000", &[]);
-    let _res = instantiate(deps.as_mut(), env.clone(), info, msg).unwrap();
-
-    let info = mock_info("owner0000", &[]);
-    let msg = ExecuteMsg::RegisterMerkleRoot {
-        merkle_root: test_data.root,
-        expiration: Some(Expiry::AtTime(Milliseconds::from_nanos(100_000_000))),
-        total_amount: Some(Uint128::new(10000)),
-    };
-    execute(deps.as_mut(), env.clone(), info, msg).unwrap();
-
-    // Claim some tokens
-    let msg = ExecuteMsg::Claim {
-        amount: test_data.amount,
-        stage: 1u8,
-        proof: test_data.proofs,
-    };
-
-    let info = mock_info(test_data.account.as_str(), &[]);
-    env.block.time = Timestamp::from_nanos(100_000_000 - 1);
-    let _res = execute(deps.as_mut(), env.clone(), info, msg).unwrap();
-
-    // makes the stage expire
-    env.block.time = Timestamp::from_nanos(100_000_000 + 1);
-
-    // Can burn after expired stage
-    let msg = ExecuteMsg::Burn { stage: 1u8 };
-
-    let info = mock_info("owner0000", &[]);
-    let res = execute(deps.as_mut(), env, info, msg).unwrap();
-
-    let expected = SubMsg::new(CosmosMsg::Bank(BankMsg::Burn {
-        amount: vec![Coin {
-            amount: Uint128::new(9900),
-            denom: MOCK_NATIVE_DENOM.to_string(),
-        }],
-    }));
-    assert_eq!(res.messages, vec![expected]);
-
-    assert_eq!(
-        res.attributes,
-        vec![
-            attr("action", "burn"),
-            attr("stage", "1"),
-            attr("address", "owner0000"),
-            attr("amount", Uint128::new(9900)),
-        ]
-    );
-}
+// use andromeda_fungible_tokens::airdrop::{
```

Comment:
Should these be commented?

Reply:
Still have some commented tests because they were failing